rest/quotes/quotes.go

```diff
@@ -39,30 +35,30 @@ func (it *QuotesIter) Quote() *models.Quote {
 //   }
 //
 //   for iter.Next() {
-//       // Do something with the current value
+//       // do something with the current value
 //       log.Print(iter.Quote())
 //   }
 //   if iter.Err() != nil {
 //       return err
 //   }
 func (c *Client) ListQuotes(ctx context.Context, params models.ListQuotesParams, options ...client.Option) (*QuotesIter, error) {
-	iter, err := c.NewIter(ctx, models.ListQuotesPath, params, func(url string) (client.ListResponse, []interface{}, error) {
-		res := &models.QuotesResponse{}
-		err := c.Call(ctx, http.MethodGet, url, nil, res, options...)
-
-		results := make([]interface{}, len(res.Results))
-		for i, v := range res.Results {
-			results[i] = v
-		}
-
-		return res, results, err
-	})
+	url, err := c.EncodeParams(models.ListQuotesPath, params)
```

Comment:
i used a method like this in the generic iter branch and i think it's a bit cleaner than the way i did things before (i didn't like that NewIter was a method implemented on the client type)